example/demo/tsconfig.json

```diff
@@ -1,9 +1,5 @@
 {
-	"extends": "../../tsconfig.json",
-	"compilerOptions": {
-		"jsx": "react-jsx",
-    "jsxImportSource": "@rise-tools/react"
-	},
+	"extends": "@rise-tools/preset-typescript",
```

Comment:
Just out of curiosity, shouldn't it be `@rise-tools/preset-typescript/tsconfig`, or would it work anyway since there's only one file listed in `preset-typescript` package.json

Reply:
I initially thought the same, but after reading TypeScript documentation, it appears that if there's top-level tsconfig.json, it is automatically loaded when you refer to it this way.

Reply:
Yea, was curious becuse of the default `extend` in expo, but that's not a explicitly defined `tsconfig.json` 
```json
"extends": "expo/tsconfig.base"
```
Good to know !